src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc

```diff
@@ -16,14 +16,14 @@ This section describes how to import a root certificate authority into the brows
 .Procedure
 
 . On the left side of the Chrome browser URL address bar panel, click the icon of the `Lock`, which represents the SSL/TLS status.
++
+This icon indicates the information about your connection.
```

Comment:
```suggestion
. On the left side of the Chrome browser URL address bar panel, click the `Lock` icon, which represents the SSL/TLS status.
+
This will indicate information about your current connection.
```